concourse/pipelines/template/bosh-pipeline.yml.erb

```diff
@@ -245,6 +245,7 @@ resources:
   source:
     user: <%= info['repository'].split('/').first %>
     repository: <%= info['repository'].split('/').last %>
+    access_token: ((cf-ops-automation-git-hub-token))
```

Comment:
I'd prefer a different naming like 'bot-github-access-token' or 'github-hosted-boshrelease-access-token', as it is a token to increase queries  on github, not to access coa repo